ui/src/components/poddetail/podlogs/PodLogs.tsx

```diff
@@ -209,9 +212,9 @@ export function PodLogs({ namespaceId, podName, containerName }: PodLogsProps) {
           borderRadius: "4px",
         }}
       >
-        {filteredLogs.map((l: string) => (
+        {filteredLogs.map((l: string, idx) => (
           <Box
-              key={l}
+            key={idx}
```

Comment:
idx might be too generic. It might class with other idx. Can we do `${l}-${idx}`. will this work